programs/address-lookup-table/src/processor.rs

```diff
@@ -25,7 +23,7 @@ declare_process_instruction!(Entrypoint, DEFAULT_COMPUTE_UNITS, |invoke_context|
     let transaction_context = &invoke_context.transaction_context;
     let instruction_context = transaction_context.get_current_instruction_context()?;
     let instruction_data = instruction_context.get_instruction_data();
-    match limited_deserialize(instruction_data)? {
+    match limited_deserialize(instruction_data, solana_packet::PACKET_DATA_SIZE as u64)? {
```

Comment:
Why has this function changed?

Reply:
this was calling the wrapper function in solana-sdk: https://github.com/anza-xyz/agave/blob/d303032418b01dc037427aed0b6ea850a7a0b53e/sdk/src/program_utils.rs#L9

I replaced it with the inner function which has the same name and takes a size parameter